core/consensus/babe/impl/babe_synchronizer_impl.cpp

```diff
@@ -119,7 +119,12 @@ namespace kagome::consensus {
                 return requested_blocks_handler(blocks_opt.value());
               }
             }
-            self->logger_->warn("Could not sync");
+            else if (not response_res) {
```

Comment:
Not related to this PR, but I am too lazy to introduce another PR because of that changes